lib/aws/vpc/index.ts

```diff
@@ -0,0 +1,127 @@
+import * as aws from '@cdktf/provider-aws';
```

Comment:
Please import each class separately, e.g. `import { Vpc } from '@cdktf/provider-aws/lib/vpc'` and change all the usages